Backpack/StarRating/Classes/BPKStar.m

```diff
@@ -144,7 +144,7 @@ - (UIColor *)currentStarColor {
 - (BPKIconSize)iconSizeForStarSize:(BPKStarSize)size {
     switch (size) {
     case BPKStarSizeXLarge:
-        return BPKIconSizeXLarge;
+        return BPKIconSizeLarge;
```

Comment:
Should we support the XL use-case? 
Right now setting XL to Large would not produce the wanted outcomes?